src/datasource.ts

```diff
@@ -329,20 +329,96 @@ export default class GoogleStackdriverDatasource {
     });
   }
 
-  getMetricLabel(aliasPattern, series) {
-    let aliasRegex = /\{\{(.+?)\}\}/g;
+  filterSeries(target, response) {
+    if (!_.has(target, 'seriesFilter') ||
+        target.seriesFilter.mode === 'NONE' ||
+        target.seriesFilter.type === 'NONE' ||
+        target.seriesFilter.param === '') {
+      return response;
+    }
+
+    let param = _.toNumber(target.seriesFilter.param);
+    if (_.isNaN(param)) return response;
```

Comment:
For what it's worth you can check this before you try to parse it (it works on strings).

Reply:
lodash seems to be working differently on strings, so I'll leave it as is.